cmd/geth/main.go

```diff
@@ -348,10 +348,11 @@ func prepare(ctx *cli.Context) {
 			log.Info("Bumping default cache on mainnet", "provided", ctx.Int(utils.CacheFlag.Name), "updated", 4096, "network", ctx.String(utils.OPNetworkFlag.Name))
 			ctx.Set(utils.CacheFlag.Name, strconv.Itoa(4096))
 		}
-	} else if ctx.String(utils.SyncModeFlag.Name) != "light" && !ctx.IsSet(utils.CacheFlag.Name) && ctx.IsSet(utils.OpBNBMainnetFlag.Name) {
-		// we're really on opBNB mainnet. Bump that cache up
-		log.Info("Bumping default cache on mainnet", "provided", ctx.Int(utils.CacheFlag.Name), "updated", 4096, "network", ctx.String(utils.OpBNBMainnetFlag.Name))
-		ctx.Set(utils.CacheFlag.Name, strconv.Itoa(4096))
+	} else if ctx.String(utils.SyncModeFlag.Name) != "light" && !ctx.IsSet(utils.CacheFlag.Name) &&
+		(ctx.IsSet(utils.OpBNBMainnetFlag.Name) || ctx.IsSet(utils.OpBNBTestnetFlag.Name)) {
+		// we're really on opBNB network. Bump that cache up
+		log.Info("Bumping default cache on opBNB", "provided", ctx.Int(utils.CacheFlag.Name), "updated", 32000)
+		ctx.Set(utils.CacheFlag.Name, strconv.Itoa(32000))
```

Comment:
32000 seems too high, currently our mainnet node appears to be at 22000, can you confirm again?

Reply:
Yes, our doc is still 32000, but i think 22000 is more reasonable. Updated, pls confirm it.